src/app/zap-templates/zcl/data-model/chip/scene.xml

```diff
@@ -48,14 +57,14 @@ limitations under the License.
     <server tick="false" init="false">true</server>
     <globalAttribute side="either" code="0xFFFD" value="5"/>
    
-    <attribute side="server" code="0x0000" define="SCENE_COUNT" type="int8u" min="0x00" max="0xFF" writable="false" default="0x00" optional="false">SceneCount</attribute>
-    <attribute side="server" code="0x0001" define="CURRENT_SCENE" type="int8u" min="0x00" max="0xFF" writable="false" default="0x00" optional="false">CurrentScene</attribute>
-    <attribute side="server" code="0x0002" define="CURRENT_GROUP" type="group_id" min="0x0000" max="0xFFF7" writable="false" default="0x0000" optional="false">CurrentGroup</attribute>
-    <attribute side="server" code="0x0003" define="SCENE_VALID" type="boolean" min="0x00" max="0x01" writable="false" default="0x00" optional="false">SceneValid</attribute>
+    <attribute side="server" code="0x0000" define="SCENE_COUNT" type="int8u" min="0x00" max="0xFF" writable="false" default="0x00" optional="true">SceneCount</attribute>
```

Comment:
It's not just optional.  It's literally not allowed in Matter, right?

I filed https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/7910 to clean up the spec here.

If we do decide to remove these from XML, that needs to happen in a separate PR, so this is ok for now,  but it's going to _really_ confuse ZAP users.

Reply:
Opened https://github.com/project-chip/connectedhomeip/issues/30256